src/Skiplist.hs

```diff
@@ -126,6 +126,9 @@ nameList =
     eq
       "gitlab-workhorse"
       "@globin asked to skip in https://github.com/NixOS/nixpkgs/pull/52286#issuecomment-447653409",
+    eq
+      "gitlab-elasticsearch-indexer"
+      "@yayayayaka asked to skip in https://github.com/NixOS/nixpkgs/pull/244074#issuecomment-1641657015"
```

Comment:
Missing a comma.

https://github.com/ryantm/nixpkgs-update/pull/361

Reply:
Oh, sorry. Thanks for spotting this :heart: